scripts/aws_helper.py

```diff
@@ -1,11 +1,12 @@
 import json
 from os import environ
+from typing import Any
 
 import boto3
 from linz_logger import get_log
 
 
-class Credentials:
+class Credentials:  # pylint: disable=too-few-public-methods
```

Comment:
Not necessary for this PR (and I've created this class...), but it would be better to use a `namedtuple` instead of a `class`. Feel free to do it or not.